templates/customize-grid/grid-levels.html

```diff
@@ -0,0 +1,47 @@
+<div id="level">
+  <table id="grid-table">
+    <thead>
+      <tr>
+        <th>Students</th>
+        {% for adventure in class_adventures[level] %}                   
+          <th>{{ adventure }}</th>
+        {% endfor %}
+      </tr>
+    </thead>
+    <tbody>
+      {% for row in matrix_values %}
+        <tr>
+          {% set student_index = loop.index0 %}
+          {% set student = class_info['students'][student_index] %}
+          <th>{{ student }}</th>
+          {% for checkbox in row %}
+            {% set adventure_index = loop.index0 %}
+            {% set adventure_id = adventure_names[class_adventures[level][adventure_index]] %}
+            {% set student_adventure_id = student ~ '-' ~ adventure_id ~ '-' ~ level %}
+            {% if checkbox == 0 %}
+              <th class="table-header">
+                <div>
+                  <input type="checkbox" name="myCheckbox" class="checkbox-wrapper" value="checked" hx-post="/grid_overview/class/{{ class_info.id }}/level?level={{ level }}&student_index={{ student_index }}&adventure_index={{ adventure_index }}" hx-target="#level" hx-trigger="change">
```

Comment:
Can you also give change the name `myCheckbox` for something more descriptive, here and anywhere else it appears.